include/onnxruntime/core/framework/kernel_def_builder.h

```diff
@@ -64,6 +65,10 @@ class KernelDef {
     return alias_map_;
   }
 
+  const nonstd::optional<std::pair<int, int>>& VariadicAlias() const {
+    return variadic_alias_offsets_;
```

Comment:
>variadic_alias_offsets_ [](start = 11, length = 23)

How would it be used to represent Lamb's schema such as [eta, w0, g0, m0, v0, w1, g1, m1, v1] -> [w0, m0, v0, w1, g1, v1]? I think we need an initial bias and a vector to specify the input-to-output mapping in a peridic pattern. For my example here, the vector is [0, -1, 1, 2] where '-1' mean 'g0' is not mapped to output.

Reply:
lamb is too special a case, I will leave it out, to make the interface clean and simple.

Reply:
Many optimizers may fall into that category, not only Lamb.